httpChecker/httpChecker.go

```diff
@@ -34,6 +34,7 @@ func (h *HttpCheck) Check() (bool, bool, error) {
 	if err != nil {
 		return false, true, err
 	}
+	defer resp.Body.Close()
```

Comment:
Nice catch, I forgot that in the first place.
Also, since the app will exit either after a successful response or after the timeout. The issue is not severe.